__tests__/src/diff-snapshot.spec.js

```diff
@@ -187,6 +189,7 @@ describe('diff-snapshot', () => {
       });
 
       expect(mockWriteFileSync).toHaveBeenCalledWith(path.join(mockSnapshotsDir, `${mockSnapshotIdentifier}-snap.png`), mockImageBuffer);
+      expect(mockWriteFileSync).toHaveBeenCalled();
```

Comment:
you can move this up to being called before the existing assertion and may want to specify that it was only called once.